examples/10.dynamic_configuration/README.md

```diff
@@ -0,0 +1,125 @@
+Safe Configuration Management
+=============================
+
+This example shows how dynamic configuration changes can be made to compartments using the ChERIoT features of static sealed capabilities, memory claims, a futex, and a sandbox compartment for validating untrusted data.
+
+In this model a configuration item is a named blob of data. There are compartments which supply configuration items (for example received by them via the network) and other compartments that need to receive specific configuration items when ever they change. None of these compartments know about or trust each other; keeping them decoupled helps to keep the system simple, and make it easy to add new items and compartments with a minimal amount of development.
```

Comment:
We try to follow a convention in Markdown of one sentence per line.  It makes it easier to review diffs because changing one sentence doesn't affect the rest of the paragraph.